src/Widgets/DataLabelWidget/RelativeDate.tsx

```diff
@@ -0,0 +1,57 @@
+import { formatDistance } from 'date-fns'
```

Comment:
* [ ] FYI I see that we have a lot of date logic in this module anyway. So we could also use the JS built-in relative helpers instead of adding a dependency.

See https://github.com/infopark/scrivito.com/pull/1823 for an example.